components/user_settings/notifications/user_settings_notifications.jsx

```diff
@@ -141,6 +141,8 @@ export default class NotificationsTab extends React.PureComponent {
         super(props);
 
         this.state = getNotificationsStateFromProps(props);
+        this.customCheck = React.createRef();
+        this.customMentions = React.createRef();
```

Comment:
in order to be consistent with the rest of PRs, could you rename the references?
```suggestion
        this.customCheckRef = React.createRef();
        this.customMentionsRef = React.createRef();
```